src/Web/Rsp.IrasPortal.Web/Controllers/ApplicationController.cs

```diff
@@ -247,6 +247,18 @@ public async Task<IActionResult> MyApplications()
         return this.ServiceError(applicationServiceResponse);
     }
 
+    public IActionResult ProjectOverview()
+    {
+        var model = new ProjectOverviewModel
+        {
+            ProjectTitle = TempData["ProjectTitle"] as string ?? string.Empty,
+            CategoryId = TempData["CategoryId"] as string ?? string.Empty,
+            ApplicationId = TempData["ApplicationId"] as string ?? string.Empty
```

Comment:
I forgot to mention that please use define these keys in `TempDataKeys` class as constants and follow the naming convention. This way you only define it once and reuse wherever needed, instead of hardcoding.
```suggestion
            ProjectTitle = TempData[TempDataKeys.ProjectTitle] as string ?? string.Empty,
            CategoryId = TempData[TempDataKeys.CategoryId] as string ?? string.Empty,
            ApplicationId = TempData[TempDataKeys.ApplicationId] as string ?? string.Empty
```